src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,304 @@
+package duke;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.EmptyDescriptionException;
+import duke.exceptions.InvalidKeywordException;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.ToDo;
+
+
+import java.util.ArrayList;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Duke {
+
+    public static final String HORIZONTAL_LINE = "____________________________________________________________";
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static int taskCount = 0;
+    private static String userInputLine;
+    private static String command;  // one question: when to make those variable static?
+    private static String description;
+    private static String taskName;
+    private static String taskTime;
```

Comment:
These variables are in a way acting like global variables. Is there a way we can avoid this?